### PR TITLE
Change code owners

### PR DESCRIPTION
**Pull request by kjhman21**
## Proposed changes

`.github/CODEOWNERS` is updated.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

N/A

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -24,48 +24,36 @@
...
-/cmd/kscn/         @KimKyungup
+/api/              @aidan-kwon
+/blockchain/       @ehnuje @nogang @aidan-kwon @kjhman21
+/build/            @KimKyungup @jeonghonah
+/client/           @KimKyungup @jeonghonah
+/cmd/              @KimKyungup @jeonghonah
 /common/           @andybclee
 /consensus/        @andybclee
 /console/          @andybclee
 /contracts/        @andybclee
```
Comment (KimKyungup) [inline .github/CODEOWNERS]: @kjhman21 Could you add @KimKyungup @jeonghonah to contract?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -24,48 +24,36 @@
...
-/.github/          @kjhman21
 /accounts/         @kjhman21
-/api/              @kjhman21
-/blockchain/       @kjhman21
-/build/            @bitaholic
-/client/           @KimKyungup
-/cmd/              @bitaholic
-/cmd/kscn/         @KimKyungup
+/api/              @aidan-kwon
+/blockchain/       @ehnuje @nogang @aidan-kwon @kjhman21
```
Comment (KimKyungup) [inline .github/CODEOWNERS]: @kjhman21 Could you add @KimKyungup to blockchain?
I would like to follow tx_list/pool/journal.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -24,48 +24,36 @@
 # subdirectories.
 #/build/logs/      @doctocat
 
-/.circleci/        @bitaholic
-/.github/          @kjhman21
 /accounts/         @kjhman21
```
Comment (KimKyungup) [inline .github/CODEOWNERS]: @kjhman21 Could add @KimKyungup to accounts?
I would like to look bind/simulated backend.

File: .github/CODEOWNERS (hunk 2 above)
Comment (jeonghonah) [reply .github/CODEOWNERS]: Could you add @jeonghoanah to blockchain?

File: .github/CODEOWNERS (hunk 3 above)
Comment (jeonghonah) [reply .github/CODEOWNERS]: Could you add @jeonghonah to accounts?

File: .github/CODEOWNERS (hunk 2 above)
Comment (kjhman21) [inline .github/CODEOWNERS]:
```suggestion
/blockchain/       @ehnuje @nogang @KimKyungup @jeonghonah 
/blockchain/types @aidan-kwon @jimni1222
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -24,48 +24,37 @@
...
```
Comment (kjhman21) [inline .github/CODEOWNERS]:
```suggestion
/accounts/         @KimKyungup @jeonghonah 
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -24,48 +24,37 @@
...
+/api/              @aidan-kwon
+/blockchain/       @ehnuje @nogang @KimKyungup @jeonghonah 
+/blockchain/types @aidan-kwon @jimni1222
+/build/            @KimKyungup @jeonghonah
+/client/           @KimKyungup @jeonghonah
+/cmd/              @KimKyungup @jeonghonah
 /common/           @andybclee
 /consensus/        @andybclee
 /console/          @andybclee
 /contracts/        @andybclee
```
Comment (kjhman21) [inline .github/CODEOWNERS]:
```suggestion
/contracts/        @KimKyungup @jeonghonah 
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -24,48 +24,37 @@
...
+/fork/             @aidan-kwon
 /governance/       @andybclee
-/kerrors/          @kjhman21
+/kerrors/          @aidan-kwon
 /log/              @ehnuje
-/metrics/          @bitaholic
-/networks/         @bitaholic
-/node/             @bitaholic
-/node/sc/          @KimKyungup @jeonghonah
+/metrics/          @ehnuje
```
Comment (kjhman21) [inline .github/CODEOWNERS]:
```suggestion
/metrics/          @nogang 
```

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -24,48 +24,37 @@
...
+/networks/         @nogang @KimKyungup @jeonghonah
+/node/             @KimKyungup @jeonghonah
 /params/           @andybclee
-/params/version.go @bitaholic
+/params/version.go @kjhman21
 /ser/              @ehnuje
 /storage/          @ehnuje
-/tests/            @kjhman21
-/utils/            @bitaholic
+/tests/            @aidan-kwon
```
Comment (kjhman21) [inline .github/CODEOWNERS]:
```suggestion
/tests/            @aidan-kwon @jimni1222 
```

File: .github/CODEOWNERS [hunk 8]
```diff
@@ -24,48 +24,37 @@
...
-/kerrors/          @kjhman21
+/kerrors/          @aidan-kwon
 /log/              @ehnuje
-/metrics/          @bitaholic
-/networks/         @bitaholic
-/node/             @bitaholic
-/node/sc/          @KimKyungup @jeonghonah
+/metrics/          @nogang 
+/networks/         @nogang @KimKyungup @jeonghonah
+/node/             @KimKyungup @jeonghonah
```
Comment (kjhman21) [inline .github/CODEOWNERS]:
```suggestion
/node/             @nogang @KimKyungup @jeonghonah
```

File: .github/CODEOWNERS [hunk 9]
```diff
@@ -24,48 +24,37 @@
...
+/build/            @KimKyungup @jeonghonah
+/client/           @KimKyungup @jeonghonah
+/cmd/              @KimKyungup @jeonghonah
 /common/           @andybclee
 /consensus/        @andybclee
 /console/          @andybclee
-/contracts/        @andybclee
+/contracts/        @KimKyungup @jeonghonah 
 /crypto/           @aidan-kwon
 /datasync/         @ehnuje
```
Comment (kjhman21) [inline .github/CODEOWNERS]:
```suggestion
/datasync/         @ehnuje @nogang 
```

File: .github/CODEOWNERS [hunk 10]
```diff
@@ -24,48 +24,37 @@
...
-/api/              @kjhman21
-/blockchain/       @kjhman21
-/build/            @bitaholic
-/client/           @KimKyungup
-/cmd/              @bitaholic
-/cmd/kscn/         @KimKyungup
+/accounts/         @KimKyungup @jeonghonah 
+/api/              @aidan-kwon
+/blockchain/       @ehnuje @nogang @KimKyungup @jeonghonah 
+/blockchain/types @aidan-kwon @jimni1222
```
Comment (kjhman21) [inline .github/CODEOWNERS]:
```suggestion
/blockchain/types  @aidan-kwon @jimni1222
```

File: .github/CODEOWNERS [hunk 11]
```diff
@@ -24,48 +24,37 @@
...
-/metrics/          @bitaholic
-/networks/         @bitaholic
-/node/             @bitaholic
-/node/sc/          @KimKyungup @jeonghonah
+/metrics/          @nogang 
+/networks/         @nogang @KimKyungup @jeonghonah
+/node/             @nogang @KimKyungup @jeonghonah
 /params/           @andybclee
-/params/version.go @bitaholic
+/params/version.go @kjhman21
```
Comment (laggu) [inline .github/CODEOWNERS]:
```suggestion
/params/version.go @kjhman21
/reward/                    @laggu
```

**Comment by kjhman21** [comment]
Dear reviewers, please take another look. This will be merged as soon as you guys approve this.

**Pull request merged**